otx/algorithms/common/adapters/mmcv/hooks/recording_forward_hook.py

```diff
@@ -39,13 +40,15 @@ class BaseRecordingForwardHook(ABC):
         module (torch.nn.Module): The PyTorch module to be registered in forward pass
         fpn_idx (int, optional): The layer index to be processed if the model is a FPN.
                                   Defaults to 0 which uses the largest feature map from FPN.
+        normalize (bool, optional): Whether to normalize the resulting saliency maps.
     """
 
-    def __init__(self, module: torch.nn.Module, fpn_idx: int = -1) -> None:
+    def __init__(self, module: torch.nn.Module, fpn_idx: int = -1, normalize=True) -> None:
```

Comment:
```suggestion
    def __init__(self, module: torch.nn.Module, fpn_idx: int = -1, normalize: bool =True) -> None:
```
How about to add type hint?